here/traffic_patterns/sql/create_routing_speed_limit.sql

```diff
@@ -0,0 +1,44 @@
+CREATE OR REPLACE FUNCTION here.create_speed_limit(_routing_table_version text)
+    RETURNS void
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE STRICT SECURITY DEFINER PARALLEL UNSAFE
+AS $BODY$
+
+DECLARE
+    output_table TEXT;
+    routing_table TEXT;
+    streets_att_table TEXT;
+
+BEGIN
+    output_table := 'routing_streets_'||_routing_table_version||'_speed_limit';
+    routing_table := 'routing_streets_'||_routing_table_version;
+    street_att_table := 'streets_att_'||SUBSTRING(_routing_table_version FROM 1 FOR 4)::text; -- for when routing_streets has a suffix, e.g. _b, _tc
+
+    EXECUTE format($$CREATE TABLE here.%I AS
+                    SELECT
+                        routing_streets.link_dir,
+                        COALESCE(
+                            CASE
+                                WHEN RIGHT(routing_streets.link_dir, 1) = 'T' AND att.to_spd_lim > 0 THEN att.to_spd_lim
+                                WHEN RIGHT(routing_streets.link_dir, 1) = 'F' AND att.fr_spd_lim > 0 THEN att.fr_spd_lim
+                                ELSE NULL
+                            END, 50
```

Comment:
`COALESCE(CASE [...]  ELSE NULL END, 50)`, the last part feels redundant

Reply:
got rid of COALESCE